tests/notice_build_tests.py

```diff
@@ -31,7 +31,10 @@ def test_build_notice(self):
             'type': 'Rule',
             'volume': 66,
         }
-        self.assertEqual(build.build_notice('5', '9292', fr), [{
+        actual_notice = build.build_notice('5', '9292', fr)[0]
```

Comment:
Mind adding an assertion about the result being length 1 to match the previous semantics?